### PR TITLE
Fix dashboard access on nginx

### PR DESCRIPTION
**Pull request by realcharmer**
https://progress.opensuse.org/issues/180002

Change the default nginx configuration so the dashboard can be accessed from `localhost` as well as an IP address. The test coverage is also expanded to include all options for accessing the dashboard.

Verification run: https://openqa.opensuse.org/tests/5035573

### CODE REVIEW

File: tests/openQA/dashboard.pm [hunk 1]
```diff
@@ -8,8 +8,11 @@ sub run {
     prepare_firefox_autoconfig;
     switch_to_x11;
     ensure_unlocked_desktop();
+    start_gui_program('firefox http://localhost', 60, valid => 1);
+    assert_screen 'openqa-dashboard', 60;
```
Comment (realcharmer) [inline tests/openQA/dashboard.pm:12]: Are the delays here fine? I reckon this could be optimized further.

Comment (kalikiana) [reply tests/openQA/dashboard.pm:12]: Note that this is 60s max in the worst case, not a delay.

Comment (okurz) [reply tests/openQA/dashboard.pm:12]: I suggest to not even do those additional checks here by starting firefox as that is rather expensive. Just revert the code to check for localhost. If that works then we can be sure that also 127.0.0.1 and [::1] works. We should focus the openQA-in-openQA tests to test usual use cases, not cover all corner cases.

File: tests/install/openqa_webui.pm [hunk 2]
```diff
@@ -28,6 +28,7 @@ sub install_from_pkgs {
     assert_script_run "/usr/share/openqa/script/configure-web-proxy $proxy_args";
     if (check_var('OPENQA_WEB_PROXY', 'nginx')) {
         assert_script_run 'systemctl disable --now apache2';
+        assert_script_run 'sed -i "/server_name/ s/localhost/_/" /etc/nginx/nginx.conf';
```
Comment (okurz) [inline tests/install/openqa_webui.pm:31]: I guess this can go into configure-web-proxy, see L28

Shouldn't https://github.com/os-autoinst/openQA/blob/master/script/configure-web-proxy#L49 already do that? Maybe just this regex broke? @asdil12 introduced that initially with https://github.com/os-autoinst/openQA/commit/4fd97ce7237a45bbd83af7d874f54447903fbc38#diff-971564c2eee0f87443db77101fd84c3eb432ce45547d44178d1f1cae179ab83aR24-R33

**Review by okurz — CHANGES_REQUESTED** [review]

**Comment by realcharmer** [comment]
Alright, I took the wrong approach. I'll close this and create new PR after I fix the core issue. Thanks for the suggestions.

**Comment by b10n1k** [comment]
I run the VR and checked the audit.log, which still shows `denial`. shouldnt this fix the selinux permissions?

**Comment by okurz** [comment]
> I run the VR and checked the audit.log, which still shows `denial`. shouldnt this fix the selinux permissions?

No. This has nothing to do with SELinux
